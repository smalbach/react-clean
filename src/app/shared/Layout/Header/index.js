import React from "react";

const Header = () => (
  <div className="header">
    <h1 className="title" data-cy="title">
      Order App
    </h1>
  </div>
);

export default Header;
