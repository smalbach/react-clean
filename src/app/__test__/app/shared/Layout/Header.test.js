import React from "react";
import { mount, shallow } from "enzyme";
import { create } from "react-test-renderer";
// eslint-disable-next-line jest/no-mocks-import
import ProviderMock from "../../../../__mocks__/ProviderMock";

import Header from "../../../../shared/Layout/Header";

describe("<Header>", () => {
  const header = shallow(
    <ProviderMock>
      <Header />
    </ProviderMock>
  );

  test("Header render", () => {
    expect(header.length).toEqual(1);
  });
  test("Title render", () => {
    const header_mount = mount(
      <ProviderMock>
        <Header />
      </ProviderMock>
    );
    expect(header_mount.find(".title").text()).toEqual("Order App");
  });
});
