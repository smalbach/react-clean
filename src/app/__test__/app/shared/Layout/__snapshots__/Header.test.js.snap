// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshop  Test UI Component Header 1`] = `[Function]`;

exports[`Header snapshop UI Component Header 1`] = `[Function]`;
